config/materials/aisi_630.mtl

```diff
@@ -0,0 +1,86 @@
+﻿ND_RelParSet_K01 = {
+
+Name = AISI_630
+
+PARAMETERS =
+{
+  Name = PTC_MATERIAL_DESCRIPTION
+  Type = String
+  Default = 'Stainless steel with excellent mech props, cond. H 1075. Aka 17-4PH.
```

Comment:
@Lawproto usually we purchase the material in `Condition A`. See as a reference this issue https://github.com/icub-tech-iit/cad-mechanics/issues/34 and the associated brochure. 
I would keep the post-processing heat treatments for the dedicated section in the drawing frame.

Reply:
Ok, I am assuming the A stands for Annealed.